src/Eloquent/Concerns/SoftDeletesModels.php

```diff
@@ -61,13 +62,14 @@ protected function findQuery($resourceId)
     protected function fillAttributes($record, Collection $attributes)
     {
         $field = $this->getSoftDeleteField($record);
+        $attributesArr = $attributes->toArray();
```

Comment:
Does it not work if you just use `$attributes`? Would prefer not to have to assign to `$attributesArr` unless it is totally necessary!

Reply:
Agreed. Works on Object (Collection) for `get` and `has` so I could just use `$attributes` for that, but won't work for `forget` (or `except`). We're still left with a `toArray()` for that. Will check again though, to make sure. Will get back to you. Maybe mapping the collection and working on the nested array would work.

Reply:
Ah no problem - if it doesn't work for `except` then just leave the assignment. Bit annoying that it doesn't work for all 3!!